pkg/controller/migmigration/dvm.go

```diff
@@ -0,0 +1,123 @@
+package migmigration
+
+import (
+	"context"
+	"errors"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	dvmc "github.com/konveyor/mig-controller/pkg/controller/directvolumemigration"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func (t *Task) createDirectVolumeMigration() error {
+	existingDvm, err := t.getDirectVolumeMigration()
+	if err != nil {
+		return err
+	}
+	// If already created exit with no error
+	if existingDvm != nil {
+		return nil
+	}
+	dvm := t.buildDirectVolumeMigration()
+	if dvm == nil {
+		return errors.New("failed to build directvolumeclaim list")
+	}
+	client, err := t.getDestinationClient()
+	if err != nil {
+		return err
+	}
+	err = client.Create(context.TODO(), dvm)
+	return err
+
+}
+
+func (t *Task) buildDirectVolumeMigration() *migapi.DirectVolumeMigration {
```

Comment:
This needs to set owner references, and the migmigration_controller needs to use this to watch DVMs.